--- tgui-0.9/Makefile~	2007-08-01 23:11:52.000000000 +0200
+++ tgui-0.9/Makefile	2007-08-01 23:11:49.000000000 +0200
@@ -1,6 +1,7 @@
-PREFIX=/usr/local
+PREFIX=/usr
 FLAGS=
 INCLUDES=.
+CXX=
 
 all: lib examples
 
@@ -9,19 +10,20 @@
 	ranlib libtgui.a
 
 tgui.o: tgui.cpp TGUI/tgui.h
-	g++ $(FLAGS) -I$(INCLUDES) -c tgui.cpp
+	$(CXX) $(FLAGS) -I$(INCLUDES) -c tgui.cpp
 
 awidgets.o: awidgets.cpp TGUI/awidgets.h
-	g++ $(FLAGS) -c awidgets.cpp
+	$(CXX) $(FLAGS) -c awidgets.cpp
 
 examples: awexamp
 
 awexamp: libtgui.a awexamp.cpp
-	g++ $(FLAGS) awexamp.cpp -o awexamp -L. -ltgui `allegro-config --libs`
+	$(CXX) $(FLAGS) awexamp.cpp -o awexamp -L. -ltgui `allegro-config --libs`
 
 install:
-	cp -R TGUI $(PREFIX)/include
-	cp -R libtgui.a /usr/local/lib
+	cp -R TGUI $(DESTDIR)$(PREFIX)/include
+	cp -R libtgui.a $(DESTDIR)/usr/lib
+	cp -R awexamp $(DESTDIR)/usr/bin
 
 clean:
 	rm -f libtgui.a *.o awexamp
